hera_mc/weather.py

```diff
@@ -168,17 +170,21 @@ def _helper_create_from_sensors(starttime, stoptime, variables=None):
     portal_client = KATPortalClient(katportal_url, on_update_callback=None)
 
     histories = yield portal_client.sensors_histories(sensor_names, starttime.unix,
-                                                      stoptime.unix, timeout_sec=60)
+                                                      stoptime.unix, timeout_sec=120)
 
     weather_obj_list = []
     for sensor_name, history in histories.items():
-        time_vals = {}
         variable = sensor_var_dict[sensor_name]
+        value_times = []
+        sensor_values = []
```

Comment:
It shouldn't be necessary to change these to lists: the first thing that happens in `_reduce_time_vals` is a call to `np.unique`, whose output gives a sorted version of the inputs.

Can you see if you can find a way to make it work with the dict as before? As I commented in the original PR, doing an `in` test with a list is much more expensive than with a dict, and the weather data come in at such a high cadence that the performance difference might actually matter.